data/test/scenarios/events-test_filterable1.cfg

```diff
@@ -0,0 +1,193 @@
+# unit placed
+# prerecruit
+# recruit
+# exit_hex
+# enter_hex
+# moveto
+# attack
+# attacker hits
+# defender misses
+# attack end
+# attacker misses
+# defender hits
+# pre advance
+# advance
+# post advance
+# last breath
+# die
+{GENERIC_UNIT_TEST "events-test_filterable1" (
+    [event]
+        name=start
+# ensure attacker/defender hits/misses events are triggered
+        [modify_unit]
+            [filter]
+                name="Alice"
+            [/filter]
+            [effect]
+                apply_to=defense
+                replace=yes
+                [defense]
+                    flat=0
+                [/defense]
+            [/effect]
+        [/modify_unit]
+        [modify_unit]
+            [filter]
+                name="Bob"
+            [/filter]
+            [effect]
+                apply_to=max_experience
+                set=2
+            [/effect]
+            [effect]
+                apply_to=defense
+                replace=yes
+                [defense]
+                    castle=100
+                    frozen=100
+                    flat=100
+                [/defense]
+            [/effect]
+        [/modify_unit]
+# ensure side 2 attacks even with a 0% chance to hit
```

Comment:
Can a [command] tag be used that way?

Reply:
Updated to use [do_command] and [move], as well as split `last breath`/`die` into a separate test, since for whatever reason [do_command] wasn't triggering to have the units keep attacking each other after side 2's unit leveled up.